curvefs/src/metaserver/copyset/copyset_node_manager.cpp

```diff
@@ -164,10 +164,29 @@ CopysetNode* CopysetNodeManager::GetCopysetNode(PoolId poolId,
     return nullptr;
 }
 
-bool CopysetNodeManager::IsCopysetNodeExist(PoolId poolId,
-                                            CopysetId copysetId) {
+int CopysetNodeManager::IsCopysetNodeExist(
+    const CreateCopysetRequest::Copyset& copyset) {
     ReadLockGuard lock(lock_);
-    return copysets_.count(ToGroupId(poolId, copysetId)) != 0;
+    auto iter = copysets_.find(ToGroupId(copyset.poolid(),
+                                         copyset.copysetid()));
+    if (iter == copysets_.end()) {
+        return 0;
+    } else {
+        auto copysetNode = iter->second.get();
+        std::vector<Peer> peers;
+        copysetNode->ListPeers(&peers);
```

Comment:
compare peer num

peers [1, 2, 3]  peers[1, 2, 3, 4] is not same

Reply:
> compare peer num
> 
> peers [1, 2, 3] peers[1, 2, 3, 4] is not same

fixed